packages/codegen/src/types-builder.ts

```diff
@@ -158,8 +165,16 @@ export const getTypesBuilder = (
       return {
         key: "[]",
         val: `${buildTypeDefinition(storageEntry.type.value)}`,
+        opaque: '""',
```

Comment:
nit: I think `'never'` would work just as fine, and it's probably more semantical